doc/cookbook/source/examples/distance/canberra.rst

```diff
@@ -0,0 +1,39 @@
+==================
+Canberra Distance
+==================
+
+The Canberra distance between vectors P and Q is the similarity as measured by their angle.
+
+.. math::
+
+    d(\mathbf {P} ,\mathbf {Q} )=\sum _{i=1}^{n}{\frac {|P_{i}-Q_{i}|}{|P_{i}|+|Q_{i}|}}
+
+where :math:`\bf P` and :math:`\bf Q` are :math:`n` dimensional feature vectors.
+
+-------
+Example
+-------
+
+Imagine we have files with data. We create CDenseFeatures (here 64 bit floats aka RealFeatures) as
+
+.. sgexample:: canberra.sg:create_features
+
+We create an instance of :sgclass:`CCanberraMetric` by passing it :sgclass:`CDenseFeatures`.
```

Comment:
you mean something like:
We can create an instance of CanberraMetric by passing two Features objects for which features  we would like to measure the pairwise Canberra distances.